service/src/test/java/uk/gov/hmcts/reform/fpl/service/DirectionHelperServiceTest.java

```diff
@@ -1277,6 +1279,50 @@ private CaseData prepareCaseData(UUID directionId,
         }
     }
 
+    @Nested
+    class GetDirectionsToComplyWith {
+
+        @Test
+        void shouldReturnStandardDirectionOrderDirectionsWhenServedCaseManagementOrdersIsEmpty() {
+            List<Element<Direction>> sdoDirections = buildDirections(LOCAL_AUTHORITY);
+            CaseData caseData = caseDataWithSdo(sdoDirections)
+                .servedCaseManagementOrders(emptyList())
+                .build();
+
+            List<Element<Direction>> directions = service.getDirectionsToComplyWith(caseData);
+
+            assertThat(directions).isEqualTo(sdoDirections);
+        }
+
+        @Test
+        void shouldReturnCaseManagementOrderDirectionsWhenServedCaseManagementOrdersIsNotEmpty() {
+            List<Element<Direction>> cmoDirections = buildDirections(LOCAL_AUTHORITY);
+            CaseData caseData = caseDataWithSdo(buildDirections(CAFCASS))
+                .servedCaseManagementOrders(servedCaseManagementOrder(cmoDirections))
+                .build();
+
+            List<Element<Direction>> directions = service.getDirectionsToComplyWith(caseData);
+
+            assertThat(directions).isEqualTo(cmoDirections);
+        }
+
+        private CaseData.CaseDataBuilder caseDataWithSdo(List<Element<Direction>> sdoDirections) {
```

Comment:
`buildCaseDataWithSDO`

Reply:
hmm so techincally it doesn't build as it returns a builder... I am probably not going to change to caseDataBuilderWithSdo. The method signature should give you enough information